client/modules/workspace/src/SelectModal/SelectModal.module.css

```diff
@@ -29,3 +32,44 @@
   flex-direction: column;
   overflow: auto;
 }
+
+.modalTitle {
+  color: #222222;
+  font-weight: 400;
+}
+
+.dataFilesModalColHeader {
```

Comment:
Could add some margin-top to match design

Reply:
done